tools/autograd/gen_annotated_fn_args.py

```diff
@@ -20,6 +20,7 @@
     get_py_variable_methods,
     op_name,
 )
+import argparse
```

Comment:
Hm?

Reply:
Right - this codegen cannot run as standalone script without this import. And I noticed the other two codegens' main() was broken recently, as well.

@ezyang - shall we simply remove the main() from all the children scripts, as they are only supposed to be called from one of the master script? 
It might be useful for debugging purpose but we should add CI as you suggested. I moved the test script to the CI folder - what's the recommended entry point for lightweight smoke tests like this?

Reply:
IIRC, we have to run codegen before we can run stuff like clang-format/clang-tidy. Swapping the logic in those CI jobs to use this script might be a simple way to smoke test it.

I don't think it's necessarily bad to support running scripts individually; can be helpful if you're only working on codegen in one file. I kind of agree there are too many entry points today though...